src/Business Foundation/App/NoSeriesCopilot/src/Copilot/NoSeriesProposal.Table.al

```diff
@@ -0,0 +1,46 @@
+table 391 "No. Series Proposal"
+{
+    TableType = Temporary;
+
+    fields
+    {
+        field(1; "No."; Integer)
+        {
+
+        }
+        field(10; "Input Text"; Blob)
+        {
+
+        }
+    }
+
+    keys
+    {
+        key(PK; "No.")
+        {
+
+        }
+    }
+
+    procedure SetInputText(NewText: Text)
+    var
+        OutStr: OutStream;
+    begin
+        Rec."Input Text".CreateOutStream(OutStr, TextEncoding::UTF8);
+        OutStr.WriteText(NewText);
+    end;
+
+    procedure GetInputText(): Text
+    var
+        InStr: InStream;
+        Result: Text;
+    begin
+        Rec.CalcFields("Input Text");
+        Rec."Input Text".CreateInStream(InStr, TextEncoding::UTF8);
```

Comment:
This code will throw an error if Stream is not populated, which is possible since there is no check on call GetInputText neither in this procedure